cmd/av/stack_switch.go

```diff
@@ -44,6 +47,10 @@ var stackSwitchCmd = &cobra.Command{
 		for _, node := range rootNodes {
 			branchList = append(branchList, stackSwitchBranchList(repo, tx, branches, node)...)
 		}
+
+		if !isatty.IsTerminal(os.Stdout.Fd()) {
```

Comment:
What's the scenario here, where would this not work?

Reply:
For example, GitHub Actions runner will fail because it doesn't have a terminal attached / cannot allocate any tty.